### PR TITLE
fix(core): add output event to get page

### PR DESCRIPTION
**Pull request by Jeff1899**
Add output event to get page number, when component changed

### CODE REVIEW

File: core/src/components/cat-pagination/cat-pagination.tsx [hunk 1]
```diff
@@ -81,7 +86,7 @@ export class CatPagination {
               a11yLabel={i18n.t('pagination.prev')}
               icon={this.iconPrev}
               iconOnly
-              onClick={() => (this.page = this.page - 1)}
+              onClick={() => this.prevPage()}
```
Comment (fynnfeldpausch) [inline core/src/components/cat-pagination/cat-pagination.tsx:89]: as this has not been checked before either, I would just rely on the fact, that the button will be disabled. This just, call setPage here. Same goes for nextPage.

File: core/src/components/cat-pagination/cat-pagination.tsx [hunk 2]
```diff
@@ -110,6 +115,23 @@ export class CatPagination {
     return this.page === this.pageCount - 1;
   }
 
+  setPage(value: number) {
```
Comment (fynnfeldpausch) [inline core/src/components/cat-pagination/cat-pagination.tsx:118]: im not 100% sure, but you should be able to make this private.

**Comment by fynnfeldpausch** [comment]
Nice work. Please also add a console output to the index.html as stated in the ticket <3

**Review by fynnfeldpausch — CHANGES_REQUESTED** [review]

**Comment by Jeff1899** [comment]
> Nice work. Please also add a console output to the index.html as stated in the ticket <3

Ahh I thought that was just a tip to test it myself, thats why I removed it afterwards.
I hope it fits now, I'm not sure how you have imagined that

**Review by fynnfeldpausch — APPROVED** [review]

**Pull request merged**